packages/components/snackbar/src/Snackbar.tsx

```diff
@@ -105,14 +105,29 @@ export const Snackbar = forwardRef<HTMLDivElement, SnackbarProps>(
 
 Snackbar.displayName = 'Snackbar'
 
-export interface AddSnackbarArgs extends SnackbarItemValue, SnackBarItemOptions {}
+export interface AddSnackbarArgs extends SnackbarItemValue, SnackBarItemOptions {
+  /**
+   * Handler that is called when the snackbar is closed, either by the user
+   * or after a timeout.
+   */
+  onClose?: () => void
+  /**
+   * A timeout to automatically close the snackbar after, in milliseconds.
+   * @default 5000
+   */
+  timeout?: number
+  /**
+   * The priority of the snackbar relative to other snackbars. Larger numbers indicate higher priority.
+   */
+  priority?: number
```

Comment:
Can it be any number or do we want to restrict this value?

Reply:
For now priority will not be handled by our component. This type comes from `react-spectrum`, and I didn't see any reason to change it for now :+1: